config/schema.yaml

```diff
@@ -0,0 +1,375 @@
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+title: Avian-flu config schema
+description: >
+      This is the schema for the Nextstrain avian-flu phylogenetic workflow
+      <https://github.com/nextstrain/avian-flu>. The readme (viewable at that URL)
+      provides general information including how to run the workflow. This schema
+      presents the interface into the phylogenetic workflow(s).
+__aliases:
+  # We use YAML aliases to get the behaviour we want from $refs, namely we want the objects
+  # to be merged. This is not the case in draft-07 - any properties in an object with a '$ref'
+  # key are dropped.
+  generic_workflow_file: &generic_workflow_file
+    description: >
+      Relative path which is to be found in the analysis directory, the entry snakefile directory
+      (e.g. `avian-flu/gisaid`) or the workflow directory (`avian-flu`). First match is used.
+      File path which may include wildcards (see examples).
+    # NOTE: Some places where this is used provide their own description and thus this one isn't used.
+    type: string
+    # TODO: it's not clear whether spaces in filenames will work everywhere...
+  wildcard_param_property: &wildcard_param_property
+    # While the intention of the config is to distinguish between types (e.g. numbers, strings)
+    # In reality this doesn't matter _most_ of the time. We could create many such definitions to
+    # handle the combination of allowable types, but we can't pass in the type to this
+    # definition where it's used (e.g. via the $ref).
+    # Leaving this here as a potential TODO
+    oneOf:
+      - type: object
+        patternProperties:
+          "^[^_/]+/[^_/]+/[^_/]+$":
```

Comment:
Maybe a nit? I'm not sure if it matters _too_ much, but I think the intent of your wildcarding approach is that `*`, when it appears as an element, should appear alone, and that is not enforced by this regex — you'd need to replace the `[^_/]+` bits with something more like `(*|[^_/]+)`, I think?